learntools/data_viz_easy/ex1.py

```diff
@@ -5,7 +5,7 @@
 
 from learntools.core import *
 
-file_path = '/Users/alexiscook/learntools/learntools/data_viz_easy/input/museum_visitors.csv'
```

Comment:
We run everything from the raw dir.  So the data should be in `notebooks/data_viz_easy/input`. I moved it there, and now this works.  That also works when running it on kaggle.

Actually, there's a caveat about paths that suggests we should always use at least 2 data files for each exercise... even if only one dataset is used. I can tell you about that by voice.